cirq/contrib/routing/device.py

```diff
@@ -47,3 +47,30 @@ def gridqubits_to_graph_device(qubits: Iterable[cirq.GridQubit]):
 
 def _manhattan_distance(qubit1: cirq.GridQubit, qubit2: cirq.GridQubit) -> int:
     return abs(qubit1.row - qubit2.row) + abs(qubit1.col - qubit2.col)
+
+
+def nx_qubit_layout(graph: nx.Graph) \
```

Comment:
This can just take in an iterable of qubits (which our device graphs happen to be).

Reply:
This is to match the other networkx layout functions